pkg/pluginmanager/manager.go

```diff
@@ -746,23 +746,26 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 	// Log message based on different installation conditions
 	installingMsg, installedMsg, errMsg := getPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
 
-	var spinnerErr, pluginErr error
+	var spinnerErr error
```

Comment:
This variable and the `signalChannel` one can go in the loop below for simplicity.